LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
I've added license because there was no one, but not sure about the type. Feel free to pick another one which is more suitable.